README.md

```diff
@@ -113,6 +113,8 @@ export { UserContext, UserProvider };
 With our context created, and our provider component all set up, let's see how
 we can use this context data from other components.
 
+Curious about what children is? Review our React Children Lesson for a refresher! **Need hyperlink to related lesson**.
```

Comment:
I would use "are" here rather than "is" - I know you're referring to the concept, but we'll almost certainly get issues on it. For the hyperlink, you can use the GitHub repo.